dizoo/classic_control/cartpole/config/cartpole_dqn_gail_config.py

```diff
@@ -0,0 +1,62 @@
+from easydict import EasyDict
+
+cartpole_dqn_gail_config = dict(
+    exp_name='cartpole_dqn_gail',
+    env=dict(
+        collector_env_num=8,
+        evaluator_env_num=5,
+        n_evaluator_episode=5,
+        stop_value=195,
+    ),
+    reward_model=dict(
+        type='gail',
+        input_size=5,
+        hidden_size=64,
+        batch_size=64,
+        learning_rate=1e-3,
+        update_per_collect=100,
+        expert_data_path='cartpole_dqn/expert_data_train.pkl',
+        load_path='cartpole_dqn_gail/reward_model/ckpt/ckpt_last.pth.tar',
```

Comment:
Is load_path refering to the well-trained reward model you finally save? can we make this clearer? (let other users directly understanding that 'load_path' refers to the final reward model the algorithm learned)

Also, Why do we need load_path key here?

Reply:
There are 2 load_path:
- policy.load_path is the path where the state_dict of the **policy** is saved
- reward_model.load_path is the path where the state_dict of the **reward model** is saved

I will write a comment in the config file to clarify it